lib/chef/azure/service.rb

```diff
@@ -155,4 +96,140 @@ def is_running?
     end
     return false
   end
+
+  def add_or_update_interval_in_client_rb(client_rb, new_chef_service_interval)

```

Comment:
@mwrock Editing `client.rb` file is to change the `interval` on user's choice for `Windows` platform. We similarly modify the `cronjob` for `Linux` platform if `interval` change is demanded by user. `enable.rb` concern is to just configure the `Chef` for first time and then `enable` the `chef-service`, and so according to me doing all the manipulation to the `chef-service` should be the concern of `service.rb`.